src/aap_eda/tasks/analytics.py

```diff
@@ -60,9 +62,28 @@ def schedule_gather_analytics(
         )
 
 
-@job(ANALYTICS_TASKS_QUEUE)
 def reschedule_gather_analytics(
     queue_name: str = ANALYTICS_TASKS_QUEUE,
+) -> None:
+    """Reschedule the gather analytics job.
```

Comment:
Don't we need to do the same for `def schedule_gather_analytics`?
Also do we need to make line 35 `job = tasking.redis_connect_retry()(django_rq.job)` conditional based on the feature flag?

Reply:
No, `schedule_gather_analytics` is agnostic, since it finally calls under the advisory_lock `tasking.enqueue_delay` which does the proxy between rq/dispatcherd. https://github.com/ansible/eda-server/pull/1286/files#diff-8919c4e8183e21876fcf7b5ad8884b1efbb37a0e08544fba092d83c6f733adfbR585